packages/framework/src/Form/CustomerType.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CustomerType extends AbstractType
+{
+    /**
+     * @param \Symfony\Component\OptionsResolver\OptionsResolver $resolver
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver
+            ->setDefaults([
+                'mapped' => false,
+                'required' => false,
+                'attr' => [
+                    'readonly' => 'readonly',
+                ],
+                'data' => [
+                    'customer' => null,
```

Comment:
Is the `customer` key necessary? It would be less typing when directly set to `data`, and it would be consistent with `DisplayOnlyType`.

Reply:
done